packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,176 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let addrA: string
+  let addrB: string
+  let addrCForward: string
+  let addrCCrumbs: string
+  let addrCPie: string
+
+  async function setup (): Promise<void> {
+    addrA = await container.call('getnewaddress')
+    addrB = await container.call('getnewaddress')
+    addrCForward = await container.call('getnewaddress')
+    addrCCrumbs = await container.call('getnewaddress')
+    addrCPie = await container.call('getnewaddress')
+
+    const tokenaddrA = await container.call('getnewaddress')
+    const tokenaddrB = await container.call('getnewaddress')
+    const tokenAddrC = await container.call('getnewaddress')
+    await createToken(tokenaddrA, 'ANT', 100)
+    await createToken(tokenaddrB, 'BAT', 100)
+    await createToken(tokenAddrC, 'CAT', 200)
+
+    await container.call('accounttoaccount', [tokenaddrA, { [addrA]: `${100}@ANT` }])
+    await container.call('accounttoaccount', [tokenaddrB, { [addrB]: `${100}@BAT` }])
+    await container.call('accounttoaccount', [tokenAddrC, { [addrCForward]: `${49}@CAT` }])
+    await container.call('accounttoaccount', [tokenAddrC, { [addrCCrumbs]: `${3}@CAT` }])
+    await container.call('accounttoaccount', [tokenAddrC, { [addrCPie]: `${104}@CAT` }])
+    await container.generate(1)
+  }
+
+  async function createToken (address: string, symbol: string, amount: number): Promise<void> {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+
+    await container.waitForWalletBalanceGTE(amount)
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('utxostoaccount', [{ [address]: `${amount.toString()}@0` }])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+
+  function accToNum (acc: string): number {
+    return Number(acc.split('@')[0])
+  }
+
+  it('should sendTokensToAddress with selectionMode pie', async () => {
+    const accPieBefore = (await client.account.getAccount(addrCPie))[0]
```

Comment:
don't need to further specify `pie` your entire `it` section is already about pie selection mode.

```suggestion
    const amountBefore = (await client.account.getAccount(addrCPie))[0]
```

Reply:
and keep a habit of using the entire full name, especially within tests. We should have to guess what is `addr` `C` `Pie`